build-system/pr-validation.yaml

```diff
@@ -93,6 +93,24 @@ jobs:
       outputDirectory: "TestResults"
       artifactName: "net_tests_linux-$(Build.BuildId)"
 
+  - template: azure-pipeline.template.yaml
```

Comment:
Is this a temporary measure for validation on this PR or do you want us to keep this permanently in the build system?

Reply:
Permanent until we switch to .NET 10 permanently, since there are tests for CLR hooks now

Reply:
The new .NET 10 CI/CD is blinding fast, its only running a single unit test class in Akka.Tests project